python/pyspark/sql/tests/test_artifact.py

```diff
@@ -0,0 +1,94 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import unittest
+import os
+import tempfile
+
+from pyspark.sql.tests.connect.client.test_artifact import ArtifactTestsMixin
```

Comment:
Use `pyspark.sql.tests.connect.client.test_artifact.ArtifactTestsMixin` here to ensure the same testing as Spark Connect

Reply:
I think `ArtifactTestsMixin` has to be extracted to here.

Reply:
Could I address the relocation the test as a follow-up? I have some plan to restructure overall `test_artifact.py` across Classic and Connect.